frontend/src/components/visual-editor/form/inputs/triggers/PostbackInputV2.tsx

```diff
@@ -117,8 +134,68 @@ export const PostbackInput = ({ value, onChange }: PostbackInputProps) => {
         }),
     [block?.previousBlocks],
   );
+  const menuOptions = menu
+    .filter((menu) => menu.payload)
+    .map(({ payload, title }) => ({
+      id: payload as string,
+      label: title as string,
+      group: "menu",
+    }));
+  const contentOptions = useMemo(
+    () =>
+      (block?.previousBlocks || [])
+        .map((bId) => getBlockFromCache(bId))
+        .filter((b) => {
+          return (
+            b &&
+            b.options?.content?.entity &&
+            b.options.content.buttons.length > 0
+          );
+        })
+        .map((b) => b as IBlock)
+        .map((b) => {
+          const availableContents = (contents || []).filter(
+            ({ entity, status }) =>
+              status && entity === b.options?.content?.entity,
+          );
+
+          return (b.options?.content?.buttons || []).reduce((payloads, btn) => {
+            // Return a payload for each node/button combination
+            payloads.push({
+              id: btn.title,
+              label: btn.title,
+              group: "content",
+            });
+
+            return availableContents.reduce((acc, n) => {
+              acc.push({
+                id: n.title,
+                label: n.title,
+                group: "content",
+              });
+
+              return acc;
+            }, payloads);
+          }, [] as ContentPayloadOption[]);
+        })
+        .flat(),
+    [block?.previousBlocks, contents, getBlockFromCache],
+  );
   // Concat all previous blocks
-  const options = [...generalOptions, ...btnOptions, ...qrOptions];
+  const options = [
+    ...generalOptions,
+    ...btnOptions,
+    ...qrOptions,
+    ...menuOptions,
+    ...contentOptions,
+  ];
+  const existOption = options.find((e) => e.id === value);
```

Comment:
I think there's an issue with this one ...

Reply:
Thank you for the feedback
I will look for a fix now :technologist:

Reply:
Hi @marrouchi, the issue was fixed :white_check_mark: